src/controllers/post.js

```diff
@@ -1,4 +1,4 @@
-import { sendDataResponse } from '../utils/responses.js'
+import { sendDataResponse, sendMessageResponse } from '../utils/responses.js'
```

Comment:
I don't think you need the .js at the end of the imports

Reply:
It appears the rest of the app adds .js to the end of imports.